lib/foreman_remote_execution/engine.rb

```diff
@@ -107,6 +107,10 @@ class Engine < ::Rails::Engine
       end
       Bookmark.send(:include, ForemanRemoteExecution::BookmarkExtensions)
       HostsHelper.send(:include, ForemanRemoteExecution::HostsHelperExtensions)
+
+      # We need to explicitly force to load the Task model due to Rails loader
+      # having issues with resolving it to Rake::Task otherwise
+      require_dependency 'foreman_tasks/task'
```

Comment:
Other initializers run only once, so we would need to sacrifice the ForemanTasks::Task reloading, which is something I would like to avoid